src/com/testmonitor/actions/TestCases.java

```diff
@@ -78,40 +79,44 @@ public TestCase get(Integer id)
     }
 
     /**
-     * Search a test case
+     * Search through test cases.
      *
-     * @param search The search string
+     * @param search The search query
      *
      * @return A list of results
      */
-    public ArrayList<TestCase> search(String search)
+    public ArrayList<TestCase> search(String query)
     {
         List<NameValuePair> params = new ArrayList<>();
 
-        params.add(new BasicNameValuePair("query", search));
         params.add(new BasicNameValuePair("project_id", this.projectId.toString()));
 
+        params.add(new BasicNameValuePair("query", query));
+
         return TestCaseParser.parse(this.connector.get("test-cases", params));
     }
 
     /**
-     * Search a test case
+     * Search though test cases in a test suite.
      *
-     * @param search The search string
+     * @param search The search query
      *
      * @return A list of results
      */
-    public ArrayList<TestCase> search(String search, Integer testSuiteId)
+    public ArrayList<TestCase> search(String query, Integer testSuiteId)
     {
-        ArrayList<TestCase> testCases = this.search(search);
+        List<NameValuePair> params = new ArrayList<>();
+
+        params.add(new BasicNameValuePair("project_id", this.projectId.toString()));
+        params.add(new BasicNameValuePair("test_suite", testSuiteId.toString()));
 
-        testCases.removeIf(testCase -> !testCase.getTestSuiteId().equals(testSuiteId));
```

Comment:
Deze is wel belangrijk ;) je kan meerdere testcases hebben met dezelfde naam in andere suites

Reply:
Vervangen door nieuwe methode?